src/main/java/com/uvocab/domain/Vocab.java

```diff
@@ -0,0 +1,14 @@
+package com.uvocab.domain;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
```

Comment:
@Danilosimoes using the @Getter and @Setter with @Data is redundant, since the @Data "mix" this [annotations](https://projectlombok.org/features/Data)

And using @Data with @Builder is not a good approach, since the idea main idea of builder is create a immutable object, when @Data we can create a mutable object.

https://projectlombok.org/features/Builder